dotcom-rendering/src/web/layouts/LiveLayout.tsx

```diff
@@ -938,13 +958,14 @@ export const LiveLayout = ({ CAPI, NAV, format, palette }: Props) => {
 											sticky,
 											keyEventsTopMargin,
 											sidePaddingDesktop,
-											accordionBottomMargin,
 										]}
 									>
-										<KeyEventsContainer
-											format={format}
-											keyEvents={CAPI.keyEvents}
-										/>
+										<Island deferUntil="idle">
```

Comment:
If we're not using the useEffect within this component the Island can be removed.